### PR TITLE
Sqleet fts

### PR DESCRIPTION
**Pull request by fire**
@hidemat Can you review?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 [SQLite3](https://www.sqlite.org/) wrapper for Godot Engine
 
+## sqleet-fts Branch
```
Comment (fire) [inline README.md:5]: Can you modify this so that the descriptions fit with the main readme?

**Review by fire — COMMENTED** [review]
https://sqlite.org/src/wiki?name=StrictMode I was wanting to use this. Is it difficult to update sqleet?

**Comment by hidemat** [comment]
> https://sqlite.org/src/wiki?name=StrictMode I was wanting to use this. Is it difficult to update sqleet?

Hi sorry I kinda forgot about this too. I ran into some issues when exporting my test project from Godot 4 to android. I wasn't able to figure them out so I just gave up on sqlite in my godot project. 

I don't know much about sqleet except that it's built on an older version of sqlite due to some incompatibilities.

Is there something you want me to modify in my fork?

**Comment by fire** [comment]
I was stuck 6 months ago. I wasn't able to get it to work. Reopen if you have a better try.

**Pull request closed without merging**